ros1_bridge/__init__.py

```diff
@@ -659,9 +669,13 @@ def load_ros1_service(ros1_srv, rospack=None):
 
 
 def load_ros2_message(ros2_msg):
-    message_path = os.path.join(
-        ros2_msg.prefix_path, 'share', ros2_msg.package_name, 'msg',
-        ros2_msg.message_name + '.msg')
+    message_path = None
+    for message_location in ['msg', 'srv', 'action']:
+        message_path = os.path.join(
+            ros2_msg.prefix_path, 'share', ros2_msg.package_name, message_location,
+            ros2_msg.message_name + '.msg')
+        if os.path.isfile(message_path):
+            break
```

Comment:
To different type of interfaces with the same name can exist. So only allowing one here seems wrong.

Same below.

Reply:
Just to double check, this PR is specifically about stripping subfolders from interface files, not to provide any support for bridging actions, right? If that's the case, then it seems to me we don't need to expose messages generated to implement services nor messages and services generated to implement actions.